Add jaeger migration guide for release 2.13 to 2.14 (#17413)
kyma-project/kyma: docs/migration-guide-2.13-2.14.md

---
title: Migration Guide 2.13-2.14
---

Jaeger was deprecated with Kyma 2.11 and has been removed with Kyma 2.14. After upgrading to Kyma 2.14, run the script [2.13-2.14-cleanup-jaeger.sh](./assets/2.13-2.14-cleanup-jaeger.sh) to remove Jaeger.

> > **NOTE** Users who want to continue using Jaeger should deploy the [Jaeger example](https://github.com/kyma-project/examples/tree/main/jaeger). 

Reviewer: ```suggestion
> **NOTE** If you want to continue using Jaeger, deploy the [Jaeger example](https://github.com/kyma-project/examples/tree/main/jaeger). 
```
